modules/changes-to-cluster-logging-and-forwarding-in-logging-6.adoc

```diff
@@ -0,0 +1,267 @@
+     application:
+       namespaces:
+       - foo
+       - bar
+       includes:
+       - namespace: my-important
+         container: main
+       excludes:
+       - container: too-verbose
+----
+
+.6.x application input with namespace and container includes and excludes
+[source,yaml]
+----
+apiVersion: "observability.openshift.io/v1"
+kind: ClusterLogForwarder
+spec:
+  inputs:
+   - name: application-logs
+     type: application
+     application:
+       includes:
+       - namespace: foo
+       - namespace: bar
+       - namespace: my-important
+         container: main
+       excludes:
+       - container: too-verbose
+----
+
+[NOTE]
+====
+"application", "infrastructure", and "audit" are reserved words and cannot be used as names when defining an input.
+====
+
+=== Input receivers
+
+Changes to input receivers include:
+
+* Explicit configuration of the type at the receiver level.
+* Port settings moved to the receiver level.
+
+.5.x input receivers
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+spec:
+  inputs:
+  - name: an-http
+    receiver:
+      http:
+        port: 8443
+        format: kubeAPIAudit
+  - name: a-syslog
+    receiver:
+      type: syslog
+      syslog:
+        port: 9442
+----
+
+.6.x input receivers
+[source,yaml]
+----
+apiVersion: "observability.openshift.io/v1"
+kind: ClusterLogForwarder
+spec:
+  inputs:
+  - name: an-http
+    type: receiver
+    receiver:
+      type: http
+      port: 8443
+      http:
+        format: kubeAPIAudit
+  - name: a-syslog
+    type: receiver
+    receiver:
+      type: syslog
+      port: 9442
+----
+
+== Output specifications
+
+High-level changes to output specifications include:
+
+* URL settings moved to each output type specification.
+* Tuning parameters moved to each output type specification.
+* Separation of TLS configuration from authentication.
+* Explicit configuration of keys and secret/configmap for TLS and authentication.
+
+== Secrets and TLS Configuration
+
+Secrets and TLS configurations are now separated into authentication and TLS configuration for each output. They must be explicitly defined in the specification rather than relying on administrators to define secrets with recognized keys. Upgrading TLS and authorization configurations requires administrators to understand previously recognized keys to continue using existing secrets. Examples in the following sections provide details on how to configure *ClusterLogForwarder* secrets to forward to existing Red Hat managed log storage solutions.
+
+.Logging 6.x output using service accounu token
+[source,yaml]
+----
+...
+spec:
+  outputs:
+  - name: my-output
+    type: http
+    http:
+      url: https://my-secure-output:8080
+    authentication:
+      token:
+        from: serviceAccount
+    tls:
+      ca:
+        key: service-ca.crt
+        configMapName: openshift-service-ca.crt
+...
+----
+
+.Logging 6.x output authentication and TLS example
```

Comment:
Logging 6.x output authentication and TLS using secret data

Reply:
done